### PR TITLE
docs: use `vagrant-spk vm *` for up, halt, ssh, destroy

### PR DESCRIPTION
**Pull request by zarvox**
Adjusts docs to recommend the canonical commands as amended by sandstorm-io/vagrant-spk#145

### CODE REVIEW

File: docs/administering/guide.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ By default, Sandstorm installs within **/opt/sandstorm**.
 - Now is a good time to `cd /opt/sandstorm` and use `ls` take a look around!
 
 **Note:** If you are having trouble finding `/opt/sandstorm`, it could be because you need to `vagrant
-ssh` into the virtual machine that runs Sandstorm for you.
+vm ssh` into the virtual machine that runs Sandstorm for you.
```
Comment (paulproteus) [inline docs/administering/guide.md]: `vagrant-spk vm ssh` or `vagrant ssh` but not `vagrant-spk vm ssh`. I can fix in a follow-up commit before merging.

Comment (zarvox) [inline docs/administering/guide.md]: Oh, this doesn't seem to be particularly related to `vagrant-spk`, but rather using Sandstorm in production.  I'd say roll this diff back.

Comment (zarvox) [inline docs/administering/guide.md]: Fixed in ed3dacaf34cad7bd3a678bb2b0cf573a7a4f7e0f

**Comment by paulproteus** [comment]
Huge thanks for writing this. Merging & deploying now.

**Pull request merged**